cmd/zetaclientd/start.go

```diff
@@ -50,10 +51,15 @@ func start(_ *cobra.Command, _ []string) error {
 
 	SetupConfigForTest()
 
-	//Prompt for Hotkey and TSS key-share passwords
-	hotkeyPass, tssKeyPass, err := promptPasswords()
+	// Prompt for Hotkey, TSS key-share and relayer key passwords
+	titles := []string{"HotKey", "TSS", "Solana Relayer Key"}
```

Comment:
Unconditionally prompting for the `Solana Relayer Key` password seems to imply that it is not optional?

Reply:
For now, key in an empty password( `\n` ) to skip if the observer runner doesn't provide a key file.

Reply:
Ok but people with automation like `echo password\npassword\n | zetaclientd start` will be broken right?

Reply:
Yeah. We'll have to ask observers to key in additional `\n` to skip the Solana relayer key password. For testnet integration, ZetaChain hosted observer will need to import relayer key and enter password properly.